api/bot/slackbot/v1/slackbot.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto3";
+
+package clutch.bot.slackbot.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/bot/slackbot/v1;slackbotv1";
+
+import "api/v1/annotations.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/struct.proto";
+
+service SlackBotAPI {
+  rpc Event(EventRequest) returns (EventResponse) {
+    option (google.api.http) = {
+      post : "/v1/bot/slackbot/event"
+      body : "*"
+    };
+    option (clutch.api.v1.action).type = CREATE;
+  }
+}
+
+// Information on the bot user
+message Bot {
+  // bot id
+  string id = 1;
+  // if the bot is deleted
+  bool deleted = 2;
+  // bot name
+  string name = 3;
+  // timestamp of when the bot app was last updated
+  int64 updated = 4;
+  // unique identifier of the installed Slack application
+  string app_id = 5;
+  // image urls
+  map<string, string> icons = 6;
```

Comment:
It's kind of unclear what icons is scoped to? Is it possible for a bot to have icons?

Reply:
I'll add a comment that makes it clearer but the Slack API sends us something like the snippet below which refers to the display picture of the bot. I'm guessing that Slack saves a given image in a few different sizes
```
"icons": {
            "image_36": "https://...",
            "image_48": "https://...",
            "image_72": "https://..."
        }
```